docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -0,0 +1,136 @@
+
+The most relevant change in the Console’s behavior is that the project’s logical state will not be saved on a Git provider anymore. This will result in significant **performance improvement** during the main Console workflows, since contacting the Git provider implied a considerable overhead that will now be reduced.
+
+This change also means that the Console will not rely on the typical Git-based instruments anymore and will instead adopt a **new configuration management system**, while still maintaining consistency with the Console’s established workflows.
+
+With the new workflow, branches and tags become **revisions** and **versions**, underscoring that the information associated with them is not stored within a Git repository anymore. The concept of commit is not present anymore and the commit history is replaced by a **version history**: the user can create versions, similar to Git tags, to keep track of the project’s evolution.
+
+### New deployment workflow
+Another significant change is the way Console configurations are saved and deployed: Kubernetes **configurations will be generated and committed** to the Git repository only **during the deployment process**. 
+
+Before deploying the configurations to the cluster, users can **review** which workload changes are going to be deployed by opening the review modal. This operation gives the developer a clear view of what is going to be released on the runtime environment, and it is mandatory if the user is deploying on a production environment. 
+
+Moreover, in the new workflow the user will be able to **export Kubernetes configurations** by clicking on a download button.
+:::note  
+The review and export features are still in development and will be available in future releases.
+:::
+
+:::caution
+When deploying your configurations to the cluster, keep in mind that only the changes pushed to the repository’s default branch will be considered.
+:::
+
+The new deployment workflow also unlocks the possibility to adopt a **pull-based deployment strategy**, in which a GitOps tool is set up to automatically align the cluster’s state to the latest changes pushed to the Git repository. To find out more about pull-based deployment and its advantages, visit the [GitOps deployment strategy documentation page](/development_suite/deploy/gitops-based/index.md).
+
+## Activating the feature
+
+The GitOps support feature can be activated in different ways, depending on your needs. In order to access the feature, please request the support of a Mia-Platform referent.
+
+You can activate the `ENABLE_CONF_GENERATION_ON_DEPLOY` feature toggle on a new project or Company to enable the enhanced project workflow. The feature toggle activates the new versioning system and the generation of Kubernetes configurations at the time of deployment. You can enable it for a single project or for an entire Company, which means all projects in the Company will adopt the new workflow.
```

Comment:
You, or your Mia-Platform referent? In this paragraph to specify better, we can also include from where

Reply:
as of now, definitely a Mia-Platform referent; I'd highlight this in a specific banner